backend/routes/api.php

```diff
@@ -23,4 +23,10 @@
         return UserResource::make($request->user());
     });
     Route::apiResource('users', UserController::class)->only(['index']);
+
+    //Users custom routes
+    Route::prefix('users')->group(function () {
+        Route::put('info', [UserController::class, 'updateInfo']);
+        Route::put('password', [UserController::class, 'updatePassword']);
```

Comment:
add another resourceful Route for 'auth', then above it, put `TODO: add login, signup, reset_password to the controller`